app/src/main/java/chat/rocket/android/chatdetails/ui/ChatDetailsFragment.kt

```diff
@@ -197,11 +197,12 @@ class ChatDetailsFragment : Fragment(), ChatDetailsView {
                     presenter.toPinned(chatRoomId!!)
                 }
             } else {
-                if (!disableMenu) {
-                    it.addOption(getString(R.string.title_files), R.drawable.ic_files_24dp) {
-                        presenter.toFiles(chatRoomId!!)
-                    }
-                }
+            // temporarily hide files until it is fixed
```

Comment:
What do you think about the fact that in DMs, after removing Files, there's nothing at all really under Chat Details other than the name and Description (which nobody uses?)....?

Should we disable chat details entirely for DMs, and leave it in for Channels and Groups?

Reply:
my thought was that this should be temporary enough that it doesn't matter. but we can.

Reply:
I suspect it's hard to know exactly how temporary it will really be.......  if it's a better design to just remove chat details in this case, I would vote for that until it's fixed.